### ISSUE TITLE
Default config.toml not found in `0.4.0`

### ISSUE THREAD

**Issue by dbazile** [terranodo/tegola#157]
In version `0.4.0-beta` and `0.4.0-alpha`, running without the `--config=` flag can't find the default config file:

```
$ ls
config.toml  entrypoint.sh  tegola

$ ./tegola
2017/10/24 21:23:28 Loading local config (config)
2017/10/24 21:23:28 config file at location (config) not found!

$ ./tegola --config=config.toml
2017/10/24 21:23:33 Loading local config (config.toml)
2017/10/24 21:23:33 Starting tegola server on port :80
```

**Comment by ARolek** [comment]
Thanks for the report.

**Comment by ARolek** [comment]
Fixed on the v0.4.0 branch. Will be included in the new release build.
